backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/helper/LabelConstants.java

```diff
@@ -44,6 +44,8 @@ public class LabelConstants {
     public static final String SPACE = " ";
 
     public static final String SUBCLASSIFICATION = ":>";
-
-    public static final String SUBSETTING = ":>";
```

Comment:
Why do you remove this constant? SUBCLASSIFICATION & SUBSETTING are two different concepts, even if they have the same value